root/defaults/nginx/site-confs/default.conf.sample

```diff
@@ -15,16 +13,10 @@ server {
     root $root;
     index index.html index.htm index.php;
 
-    set $htpasswd_file /config/nginx/.htpasswd;
-    set $auth_basic "Restricted";
-    if (!-f $htpasswd_file) {
-        set $auth_basic off;
-    }
```

Comment:
This was clever, but overly complicated, and would not have been optional (if `.htpasswd` exited it would apply whether the user intended it or not).